templates/html/templates/common/macros.html

```diff
@@ -76,19 +76,29 @@
                         </a>
                     </div>
 
-                    <div class="collapse">
+                    <div class="collapse {{ accordion_content_state }}">
```

Comment:
and here is an if statement injecting the string show/hide based on the boolean value. Doing so also respects the encapsulation principle.